conda/recipes/libcudf/recipe.yaml

```diff
@@ -0,0 +1,317 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+# Copyright (c) 2018-2025, NVIDIA CORPORATION.
+schema_version: 1
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  minor_version: ${{ (version | split("."))[:2] | join(".") }}
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
+  cuda_major: '${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[0] }}'
+  date_string: '${{ env.get("RAPIDS_DATE_STRING") }}'
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+recipe:
+  name: libcudf-split
+
+cache:
+  source:
+    path: ../../..
+
+  build:
+    script:
+      content: |
+        cudf_ROOT="$(realpath ./cpp/build)"
+        export cudf_ROOT
```

Comment:
Is this necessary? If we remove it and it fails, that to me indicates a bug in build.sh. If it does fail, my first guess would be that the failure will be that the kafka build fails to find the libcudf build. If that is what happens, it should be possible to fix directly in build.sh. If it gets too involved we can bump to a follow up, but if that happens we should open an issue because that would definitely be a bug in build.sh

Reply:
it was not necessary